src/app/collective-rewards/rewards/components/MetricsCard.tsx

```diff
@@ -73,35 +68,60 @@ export const MetricsCard: FC<MetricsCardProps> = ({
   children,
   dataTestId,
   className,
+  ...rest
 }) => {
   const borderClasses = borderless ? '' : 'border border-white border-opacity-40 rounded-lg'
   return (
-    <div className={cn(DEFAULT_CLASSES, borderClasses, className)} data-testid={dataTestId + '_MetricsCard'}>
+    <div
+      className={cn(DEFAULT_CLASSES, borderClasses, className)}
+      data-testid={dataTestId + '_MetricsCard'}
+      {...rest}
+    >
       {children}
     </div>
   )
 }
 
-type MetricsCardTitleProps = {
-  title: string
-  'data-testid': string
-  tooltip?: TooltipProps
-}
+export type MetricsCardTitleProps =
```

Comment:
it's not clear to me why we do need to omit children

Reply:
oh , actually it is needed because TypographyProps requires children, but these props do not.

Reply:
We decided to tackle it in a separate task.